BUILDING.md

```diff
@@ -0,0 +1,189 @@
+# Building instructions
+
+## Linux
+
+Dependencies:
+
+- `cmake` >= 3.1.0
+- `zlib`
+- `qt` >= 5.5 ( >= 5.6 is required if you want to rearrange tabs using
+drag&drop)
+- `Python3.5+`
+    - `virtualenv`
```

Comment:
Should not be necessary (python 3.5 has a venv module bundled, which supersedes virtualenv)